src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -787,8 +786,8 @@ private void RefreshInstalledAndUpdatesTabs()
             {
                 await NuGetUIThreadHelper.JoinableTaskFactory.SwitchToMainThreadAsync();
 
-                _topPanel._labelInstalled.ShowWarning = false;
-                _topPanel._labelUpgradeAvailable.Count = 0;
+                _topPanel.ShowWarningOnInstalledTab(installedDeprecatedPackagesCount: 0);
```

Comment:
Perhaps "UpdateDeprecatedCountOnInstalledTab(int count)" would be better naming?   -- or something like that...Show sounds like it will always be shown.

Reply:
SetDeprecationWarningOnInstalledTab
OK?

Reply:
i prefer my suggession...feel free to ask another teammate which they prefer. the fact that count is passed in, and that when it is 0 means you are not always "setting a deprecation warning".
your proposal is an improvement over current code.

Reply:
We are thinking to leave it as I had it with this mindset:

A method named `EnableWarningIcon(bool isEnabled)` would be equivalent. I fully expect that passing a `false` here in fact disables it. My logic was... I can accept the Count argument and make the decision inside the method (essentially, `bool isWarningShown`) without the caller having to decide in advance.

Reply:
Random example I found trying to find similar naming and conventions...

Windows Forms has [Graphics.FillRectangle ](https://docs.microsoft.com/en-us/dotnet/api/system.drawing.graphics.fillrectangle?view=netframework-4.8#System_Drawing_Graphics_FillRectangle_System_Drawing_Brush_System_Drawing_Rectangle_) which takes a brush. That brush could be [Brushes.Transparent](https://docs.microsoft.com/en-us/dotnet/api/system.drawing.brushes.transparent?view=netframework-4.8). Technically this still fills the rectangle, but not visibly.... not a perfect comparison.

Fill is a nice word. More descriptive than Show. Just not sure what else to use here. Any way to integrate the word `Visibility`? ...

Reply:
Compromise: `UpdateDeprecationStatusOnInstalledTab`